hack/common.sh

```diff
@@ -33,6 +33,93 @@ get_OCP_version() {
   echo $OCP_VER_MAJOR.$OCP_VER_MINOR
 }
 
+check_git_branch(){
+  # Return if the working branch is dirty
+  if ! git diff --quiet; then
+    echo "branch dirty, exiting to not overwrite work"
+    return 1
+  fi
+}
+
+# This function creates a git branch for automating commits
+# Takes n arguments
+# 1st arg: the ID of the branch, that describes the change, such as submodule-update
+# args [2, n] arg: the versions to create
+# usage: create_git_branch_list submodule-update release-4.10 release-4.12 release-4.13
```

Comment:
The usage does not match your function description.

Reply:
I'm not sure what you mean by this :thinking:

Reply:
Just that the function description and usage together is confusing. What do you mean by `the versions to create` when you specify `release-4.10`? Isn't the `release-4.10` branch already created? From what I can tell `release-4.10` is the branch from which you want to create another branch with a set of commits. That is not clear from your description.